.github/workflows/ci-bun-transpiler.yml

```diff
@@ -18,10 +18,7 @@ jobs:
         with:
           bun-version-file: .tool-versions
       - run: yarn workspaces focus hono-middleware @hono/bun-transpiler
-      - run: yarn workspace @hono/bun-transpiler build
-      - run: yarn workspace @hono/bun-transpiler typecheck
-      - run: yarn eslint packages/bun-transpiler
-      - run: yarn prettier --check  . !packages packages/bun-transpiler
```

Comment:
Use `turbo` to run the `build` command so the build cache is used

Also removes the eslint and prettier checks as they are handled by the `build` job in `ci.yml`